babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Is there a reason we can't just use the ts compiler directly instead of indirectly with babel?

Reply:
no specific reason. Switched to tsc